src/measure_theory/l1_space.lean

```diff
@@ -805,6 +805,23 @@ by { simp [integrable.to_L1, snorm, snorm'], simp [edist_eq_coe_nnnorm_sub] }
   edist (hf.to_L1 f) 0 = ∫⁻ a, edist (f a) 0 ∂μ :=
 by { simp [integrable.to_L1, snorm, snorm'], simp [edist_eq_coe_nnnorm] }
 
+lemma tendsto_to_L1_iff_tendsto_lintegral_zero {ι} {l : filter ι} (F : ι → α → β)
+  (hF : ∀ n, integrable (F n) μ) (f : α → β) (hf : integrable f μ) :
+  tendsto (λ n, (hF n).to_L1 (F n)) l (𝓝 (hf.to_L1 f))
+    ↔ tendsto (λ n, ∫⁻ x, ∥F n x - f x∥₊ ∂μ) l (𝓝 0) :=
+begin
+  convert Lp.tendsto_Lp_iff_tendsto_ℒp' _ _,
+  ext1 n,
+  simp only [snorm_eq_snorm' one_ne_zero one_ne_top, snorm', one_to_real, one_div, rpow_one,
+    _root_.inv_one],
```

Comment:
Having seen your code, I wrote a new lemma which was merged an hour ago and can be handy here:
```suggestion
  rw snorm_one_eq_lintegral_nnnorm,
```
(you'll need to merge master)